docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  db:
+    image: postgres:alpine
+    environment:
+      - 'POSTGRES_PASSWORD=grassisevergreener'
+      - 'POSTGRES_DB=evergreen_development'
+
+  backend:
+    image: jenkinsciinfra/evergreen
```

Comment:
Shouldn't it be `jenkins/evergreen`? https://github.com/jenkins-infra/evergreen/blob/master/Makefile#L1

Reply:
OK, my bad, I understand this is the services container. How about call it `jenkinsciinfra/evergreen-services` or something? (I'm still mostly on "I don't know what I'm doing" mode, so possibly I'm missing something).